.changeset/late-hairs-kneel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-home-react': minor
+---
+
+Allow disabling the header divider on homepage cards
```

Comment:
If I'm understanding this correctly existing cards would need to be updated as well to support this? I tried to play around with these but not luck: https://github.com/backstage/backstage/blob/4de4602e941397dfd6129219dec69d89e80ff17c/packages/app/src/components/home/HomePage.tsx#L96. If I'm on the right page we should probably mention that here.

Reply:
Existing cards should not need any updates, as it's an opt-in feature. It is also not a visible change unless you're modified the theme in certain ways. Let me add some screenshots.